src/Program.cs

```diff
@@ -86,6 +89,13 @@ public static async Task<int> Run(string workspace, string verbosity, bool dryRu
             {
                 return 1;
             }
+            finally
+            {
+                if (!string.IsNullOrEmpty(currentDirectory))
+                {
+                    Environment.CurrentDirectory = currentDirectory;

```

Comment:
not sure if it is really necessary to restore the current directory when we are done.